contracts/Liquidator/Liquidator.sol

```diff
@@ -350,4 +370,50 @@ contract Liquidator is Ownable2StepUpgradeable, ReentrancyGuardUpgradeable {
             revert TreasuryPercentTooHigh(maxTreasuryPercentMantissa, treasuryPercentMantissa_);
         }
     }
+
+    function _checkForceVAILiquidate(address vToken, address borrower) private view {
+        uint256 vaiDebt_ = vaiController.getVAIRepayAmount(borrower);
+        if (!forceVAILiquidate || vaiDebt_ < minLiquidatableVAI || vToken == address(vaiController)) return;
```

Comment:
What happens if VAI liquidations are paused in Comptroller, and `forceVAILiquidate` is still set to true? Should we forbid pausing VAI liquidations if `forceVAILiquidate == true`? Or this would introduce unnecessary coupling in the code?

Reply:
@chechu AFAICS `vaiController` is not listed in comptroller which results  `checkActionPauseState` in [comptroller](https://github.com/VenusProtocol/venus-protocol/blob/f1426f07b44f9b557059a0e364ce5d90e18f997a/contracts/Comptroller/Comptroller.sol#L539)`always returns false :thinking:

Reply:
You are right, `vaiController` is not listed in Comptroller. But we use that address as a "fake" market address when we want to check if the liquidation is of a VAI position (see it [here](https://github.com/VenusProtocol/venus-protocol/blob/develop/contracts/Comptroller/Comptroller.sol#L522)). So, I assumed that if we want to pause liquidations of VAI we will use the `vaiController` address to identify VAI